### ISSUE TITLE
It's not clear which file to import for '@import "overlay/overlay"

### ISSUE THREAD

**Issue by oustn** [angular/material2#1348]
#### When I create theme file, the sass throw a error like below:

```
node_modules/@angular/material/core/_core.scss Error: It's not clear which file to import for '@import "overlay/overlay"
```
#### I try to modify the import file in the _core.scss, and it's works.

@import 'overlay/overlay.scss';

 I think maybe could change the file name of 'overlay.scss' to '_overlay.scss'
#### The material and sass version:

"@angular/material": "^2.0.0-alpha.9",
"gulp-sass": "^2.3.1",

**Comment by spock123** [comment]
As far as I understand, you should not import the overlay.scss any longer....

**Comment by wallace41290** [comment]
@spock123  I got the same error, if I understand correctly, @oustn is not trying to import overlay directly, but by importing ~@angular/material/core/theming/all-theme', that sass file is importing core.scss which is importing overlay and that is where the error is from. I just went into the core.scss and changed the import to overlay.scss to force the extension for now.

**Comment by spock123** [comment]
@wallace41290  ahh I see.

**Comment by aroop** [comment]
core/overlay folder has two files overlay.scss and overlay.css(Empty file). Deleting overlay.css fixed this issue for me.

**Comment by oustn** [comment]
@wallace41290 Yes. That's what I mean.

**Comment by qdouble** [comment]
Yeah, same issue with using scss. I had to make a script to clear css from the @angular/material directory so there would be no conflicts.

**Comment by xtianus79** [comment]
@wallace41290 Thank you +1

**Comment by stefan-leye** [comment]
+1

**Comment by jelbourn** [comment]
Fixed in #1544 which will be included in the next release.
